app/controllers/users_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  include UsersHelper
+
   before_action :authenticate_current_user, only: [:edit]
 
```

Comment:
I think you should make sure that the user is an admin to update a role.
`before_action :authenticate_admin, only: [:update_role]`